store/events.js

```diff
@@ -0,0 +1,107 @@
+            linkTo: "https://www.rainforestab.ca/edmonton.html",
+            description: ` Rainforest Alberta is an informal organization working
+                  towards improving Alberta's innovation ecosystem. They
+                  want Albertans to invent, prosper and move their ideas
+                  forward. Community members can join by signin a provided
+                  by them Social Contract. Their engagement with the
+                  Edmonton entrepreneurial community is through 3
+                  different types of events: Weekly, Monthly & Annual
+                  Summits.`,
+        },
+        elug: {
+            name: "Edmonton Linux Users Group",
+            logo: require("../assets/logos/elug.png"),
+            linkTo: "https://elug.ca",
+            description: ` The Edmonton Linux User Group (ELUG) is a group of
+                  enthusiasts interested in sharing knowledge of Linux and
+                  GNU/Linux-based operating systems, experiences (and
+                  beer?), and enjoy the freedom and adventure of using
+                  free and open source software. Everybody is welcome.
+                  They virtually meet every 4th Thursday of the month.`,
+        },
+        ohm: {
+            name: "Open Hardware Meetup",
+            logo: require("../assets/logos/ohm.png"),
+            linkTo:
+                "https://www.meetup.com/edmontonunlimited/events/pcwlzsyfchbpb/",
+            description: ` The Open Hardware Meetup is a place where all kinds of
+                  people can share an interest in the intersection of
+                  hardware, software, connectivity, and user experience.
+                  Everyone is welcome. Some of their work examples from
+                  past includes: learn to solder, basics of arduino,
+                  automated basketball robots from high school team,
+                  magnetic computer displays for the virtually impaired.
+                  They meet every 2nd Thursday of every month.`,
+        },
+        bcyeg: {
+            name: "Beta City YEG",
+            logo: require("../assets/logos/bcyeg.png"),
+            linkTo: "https://betacity.ca/",
+            description: ` Beta City YEG is Edmonton’s citizen driven meetup with a
+                  mandate to connect citizens with opportunities to learn
+                  about, provide feedback for, and participate in projects
+                  that improve their community through connection and
+                  technology. They meet every 4th Wednesday of the month.`,
+        },
+        yegTechWednesdays: {
+            name: "Edmonton Tech Wednesdays",
+            logo: require("../assets/logos/yeg-tech-wed.png"),
+            linkTo:
+                "https://www.eventbrite.ca/e/edmonton-tech-wednesdays-tickets-662082084337",
+            description: `Edmonton Tech Wednesdays is a weekly event where anyone who loves tech 
```

Comment:
I know linting is really up to our linter but why did it format the ones above this so beautifully and the ones after this so weird? If you try to format these ones like the ones above does it yell at you?

Reply:
I did not do anything special to format neither of these files. I have been using just Vetur which was the recommended extension for Vue/Nuxt2.

Reply:
Also, the linter gives me issues only when I am dealing with .vue files only. This one is a .js file.